src/snipster/models.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime, timezone
+from enum import Enum
+
+from sqlalchemy import Enum as SQLAlchemyEnum
+from sqlmodel import Field, SQLModel, create_engine
+
+
+class Language(Enum):
+    PYTHON = "python"
+    JAVASCRIPT = "javascript"
+    RUST = "rust"
+    GO = "go"
+    TYPESCRIPT = "typescript"
+    SQL = "sql"
+    PLSQL = "plsql"
+
+
+class Snippet(SQLModel, table=True):
+    id: int | None = Field(default=None, primary_key=True)
+    title: str
+    code: str
+    description: str | None
+    language: Language = Field(sa_column=SQLAlchemyEnum(Language))
```

Comment:
nice, do you get enum keys or values stored in the db here? 
see: https://pybites.circle.so/c/active-cohort-discussion-forum/cohort-call-questions#comment_wrapper_72880850